lib/graphql/resolvers.js

```diff
@@ -0,0 +1,156 @@
+    })
+}
+
+/*
+ * TODO: From `lib/routes/termsAndConditionsRoutes.js` -- remove this after
+ * terms are removed from the GraphQL API too.
+ */
+const massageTerms = terms => (terms.active && terms.text) ? ({
+  delay: Boolean(terms.delay),
+  title: terms.title,
+  text: nmd(terms.text),
+  accept: terms.acceptButtonText,
+  cancel: terms.cancelButtonText,
+}) : null
+
+const staticConfig = (parent, { currentConfigVersion }, { deviceId, deviceName, settings }, info) =>
+  Promise.all([
+    plugins(settings, deviceId).staticConfigQueries(),
+    !!configManager.getCompliance(settings.config).enablePaperWalletOnly,
+    configManager.getTriggersAutomation(settings.config),
+    buildTriggers(configManager.getTriggers(settings.config)),
+    configManager.getWalletSettings('BTC', settings.config).layer2 !== 'no-layer2',
+    configManager.getLocale(deviceId, settings.config),
+    configManager.getOperatorInfo(settings.config),
+    configManager.getReceipt(settings.config),
+    massageTerms(configManager.getTermsConditions(settings.config)),
+    !!configManager.getCashOut(deviceId, settings.config).active,
+  ])
+    .then(([
+      staticConf,
+      enablePaperWalletOnly,
+      triggersAutomation,
+      triggers,
+      hasLightning,
+      localeInfo,
+      operatorInfo,
+      receiptInfo,
+      terms,
+      twoWayMode,
+    ]) =>
+      (currentConfigVersion && currentConfigVersion >= staticConf.configVersion) ?
+      null :
+      _.flow(
+        _.assign({
+          enablePaperWalletOnly,
+          triggersAutomation,
+          triggers,
+          hasLightning,
+          localeInfo: {
+            country: localeInfo.country,
+            languages: localeInfo.languages,
+            fiatCode: localeInfo.fiatCurrency
+          },
+          machineInfo: { deviceId, deviceName },
+          twoWayMode,
+          speedtestFiles,
+          urlsToPing,
+          terms,
+        }),
+        _.update('triggersAutomation', _.mapValues(_.eq('Automatic'))),
+        addOperatorInfo(operatorInfo),
+        addReceiptInfo(receiptInfo)
+      )(staticConf))
+
+
+const setZeroConfLimit = config => coin =>
+  _.set(
+    'zeroConfLimit',
+    configManager.getWalletSettings(coin.cryptoCode, config).zeroConfLimit,
+    coin
+  )
+
+const dynamicConfig = (parent, variables, { deviceId, operatorId, pid, settings }, info) => {
+  state.pids = _.update(operatorId, _.set(deviceId, { pid, ts: Date.now() }), state.pids)
+  return plugins(settings, deviceId)
+    .dynamicConfigQueries()
+    .then(_.flow(
+      _.update('coins', _.map(setZeroConfLimit(settings.config))),
+      _.set('reboot', !!pid && state.reboots?.[operatorId]?.[deviceId] === pid),
+      _.set('shutdown', !!pid && state.shutdowns?.[operatorId]?.[deviceId] === pid),
+      _.set('restartServices', !!pid && state.restartServicesMap?.[operatorId]?.[deviceId] === pid),
+    ))
+}
+
+
+module.exports = {
+  Query: {
+    staticConfig,
+    dynamicConfig
+  }
```

Comment:
I think I would rather have a single query that returns both static and dynamic config. If you organize that way it means that you can run pollQueries before and pass it down as a parameter.